src/medkit/medkit.tsx

```diff
@@ -120,59 +146,39 @@ export const Medkit: React.FC<MedkitParams> = ({scenario, budget, medkitId}: Med
-                                                                        Remove
-                                                                    </button>
-                                                                </td>
-                                                            </tr>
-                                                        )}
-                                                    </React.Fragment>
-                                                );
-                                            })}
-                                        </>
-                                    )}
-                                </tbody>
-                                <tfoot>
-                                    <tr>
-                                        <td></td>
-                                        <td>{totalItemsPicked} Items</td>
-                                        <td>{totalItemsScore} Points</td>
-                                        <td></td>
-                                    </tr>
-                                </tfoot>
-                            </table>
+                            <div className={'row bg-dark p-3 mx-0 text-light'}>
+                                <div className="col-5">
+                                    <p className="h2">
+                                        {totalItemsPicked} Item{totalItemsPicked == 1 ? '' : 's'}
+                                    </p>
+                                </div>
+                                <div className="col-7">
+                                    <p className="h2">
+                                        {totalItemsScore} / {budget} Points
+                                    </p>
+                                    <p className={'medkit__basket--warning'}>
+                                        {/* eslint-disable-next-line max-len */}
+                                        {totalItemsScore >= budget ? 'Your selections are over budget' : ''}
+                                    </p>
+                                </div>
+                                <div className="col-12 d-flex">
+                                    <button
+                                        type={'button'}
+                                        onClick={resetSelections}
+                                        className={'btn btn-secondary ms-auto me-2'}>
+                                        Reset Selections
+                                    </button>
+                                    <a className="btn btn-danger"
+                                        href={
+                                            // eslint-disable-next-line max-len
+                                            Number(medkitId) < 3 ? `/medkit/${Number(medkitId) + 1}` : '/medkit/summary'
+                                        }>
+                                        {Number(medkitId) < 3 ? 'Next Medkit' : 'Medkit Summary'}
```

Comment:
If I'm over points...should I be allowed to move onto the Next Medkit?

Reply:
Yes, this question came up with the client. They thought it would be ok because then they could discuss why someone went over.